src/sexp2k.c

```diff
@@ -102,6 +118,46 @@ ZK from_nyi_robject(S marker, SEXP sxp){
 	return attR(kp((S)Rf_type2char(TYPEOF(sxp))),sxp);
 }
 
+ZK from_frame_robject(SEXP sxp){
+	// TODO: Convert to table
+	J length = LENGTH(sxp);
+	SEXP colNames = Rf_getAttrib(sxp, R_NamesSymbol);
+	SEXP rowValues = Rf_getAttrib(sxp, R_RowNamesSymbol);	
+	K kRowValues = from_any_robject(rowValues);
+
+	K x = ktn(0, length + 1),
```

Comment:
so `x` is values and `v` is keys?
could you rename keys to be `k` and values `v`?
`K k = ktn(KS,lenght+1)`